ktor-client/ktor-client-core/common/src/io/ktor/client/features/cookies/AcceptAllCookiesStorage.kt

```diff
@@ -39,9 +41,7 @@ class AcceptAllCookiesStorage : CookiesStorage {
         }
     }
 
-    override fun close() {
-        mutex.close()
-    }
+    override fun close() {}
```

Comment:
So coroutines' lock does pooling for mutex_t so no need to terminate it explicitly, right? 

codestyle?

Reply:
Yep. Thanks will fix